<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px</string>
    </property>
    <property name="text">
     <string>Upload Track Layout</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="GoButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectFileButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>351</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
